lib/application/index.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @fileOverview
+ * A module that provides a minimalist application class, useful for managing
+ * configuration and rendering.
+ */
+
+// --------------------------------------------------------------------------
+// Functions.
+// --------------------------------------------------------------------------
+
+/**
+ * Process a queue.
+ *
+ * @param {Mixed[]} queue An array of queued items.
```

Comment:
We haven't afaik, but if we want to, we should decide now.
Otherwise, this is a non-issue.

I'm not saying we should, but I figured it worth mentioning.